CHANGES.md

```diff
@@ -3,6 +3,7 @@
 - 2.5.3
   - Add future warning and pending deprecation for 3.0 version preparation #89
   - Add ISSUE_TEMPLATE to ask for default information on issue tracker
+  - fix #93 issue
```

Comment:
@dpizetta released 2.5.3 a few days ago. 

I think this change should at least be in a new ``2.5.4`` section. @dpizetta What do you think?

Reply:
@ColinDuquesnoy, @sawt00th I agree. Please, insert a new version 2.5.4 with this fix, remember to change the version itself in __init__.  I would ask you to add more details to your message in changelog with something like this: Fix indicator image of checkable QGroupBox, #93, so if you read it you don't need to search by the issue.

BTW, thank you!